### PR TITLE
fix: features

### PR DESCRIPTION
**Pull request by weichweich**
## Checklist:

- [ ] I have verified that the code works
  - [ ] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: scripts/all.fish [hunk 1]
```diff
@@ -0,0 +1,11 @@
+for features in "--all-features" "--features runtime-benchmarks" "--features try-runtime"
```
Comment (wischli) [inline scripts/all.fish:1]: When I try to execute this, I receive an `error: no such subcommand: `workspaces`.

**Review by wischli — APPROVED** [review]
LGTM! However, I cannot execute the fish script.

File: runtimes/common/Cargo.toml [hunk 2]
```diff
@@ -30,6 +30,9 @@ sp-std = {git = "https://github.com/paritytech/substrate", default-features = fa
 default = ["std"]
 fast-gov = []
 runtime-benchmarks = [
+  "attestation/runtime-benchmarks",
+  "frame-support/runtime-benchmarks",
```
Comment (ntn-x2) [inline runtimes/common/Cargo.toml:34]: Why does runtime common enable these features upon runtime benchmarks? I'm talking specifically about `attestation` and `parachain-staking`.

File: scripts/all.fish (hunk 1 above)
Comment (ntn-x2) [inline scripts/all.fish:1]: Would it also make sense to test with no (default) features, and with the mock feature, where specified? Anyway very good solution to the problem!

**Review by ntn-x2 — COMMENTED** [review]
Just two small questions, but otherwise LGTM!

File: scripts/all.fish (hunk 1 above)
Comment (weichweich) [reply scripts/all.fish:1]: You have to install `cargo-workspaces`. I will add a comment for that.

Comment (weichweich) [reply scripts/all.fish:5]: yes that's a good idea

Comment (weichweich) [reply scripts/all.fish:5]: default features are a good idea. But mock doesn'T work since not all of the crates have that feature

**Pull request merged**